azure-pipelines.yml

```diff
@@ -1,6 +1,7 @@
 variables:
-  ITKGitTag: v5.1rc02
-  CMakeBuildType: MinSizeRel
+  ITKGitTag: v5.1rc01
+  ITKPythonGitTag: v5.1rc01
```

Comment:
why rc01?  This is fine, but I was expecting to be testing against the newer version.